libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -66,10 +66,9 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
 
     // @Param: TEMP
     // @DisplayName: ground temperature
-    // @Description: calibrated ground temperature in degrees Celsius
+    // @Description: User provided ground temperature in degrees Celsius. This can be used to provide a more accurate ambient air temperature for calculating the altitude the vehicle is at. This will be reset every time the vehicle is rebooted. A value of 0 means use the internal measurement of air temperature
```

Comment:
User provided ambient temperature in degrees Celsius at ground level. This is used to provide a more accurate ambient temperature to improve the calculation of the altitude the vehicle is at. This parameter is not persistent and is reset to zero every time the vehicle is rebooted. With a zero (0) value ArduPilot will use the internal measurement of ambient temperature